kernel/kernel-api/src/main/java/io/delta/kernel/Table.java

```diff
@@ -49,4 +49,16 @@ static Table forPath(TableClient tableClient, String path)
      */
     Snapshot getLatestSnapshot(TableClient tableClient)
         throws TableNotFoundException;
+
+    /**
+     * Get the snapshot at the given {@code versionId}.
+     * TODO: add information about which errors it throws?
```

Comment:
can you comment that information on this PR? and then we can decide what to include